grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -1277,7 +1277,8 @@ class QueryService implements Status {
                 ' from storage_document d ' +
                 ' where d.id not in (select distinct storage_document_id from submission_document) ' +
                 ' and d.project_key = :projectKey' +
-                ' and d.deleted = 0 '
+                ' and d.deleted = 0 ' +
+                ' and d.consent_collection_link_id is null'
```

Comment:
Conceptually, this looks like the wrong approach. A document, if it is attached to a project, should be returned even if it's also a consent group document. This should only work if the UI is explicit about what it's doing. From an API perspective, this is a very strange restriction to apply at the query level. 

If we need a method that says, give me my documents that are **NOT** in a consent group, then that should be a different method with a much clearer name that describes it as such. Additionally, that query should be called from an API that is clear, i.e., getNonConsentGroupProjectDocuments` or something like that.